apps/hashdotdev/src/_pages/docs/get-started/01_setup.mdx

```diff
@@ -0,0 +1,18 @@
+# Setting HASH up
+
+## On the global network
+
+The easiest way to get started with HASH, or to try it out, is to use the hosted instance at hash.ai
+
+You can sign up for an account at **hash.ai/signup**
```

Comment:
Should we be pointing people to `app.hash.ai` here?

Reply:
Just to make sure you're aware, I've just pasted the contents from [this notion doc](https://www.notion.so/hashintel/Docs-pages-330c115876b146f19f61ff48f8fc52d4) into the files for now to have them populated with something. It make might more sense to leave comments on their content there, as I believe that's where @nonparibus is writing them.